fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -590,16 +591,22 @@ class BrowserRobot {
         }
     }
 
-    fun verifyCookiesProtectionHintIsDisplayed(isDisplayed: Boolean) {
-        assertItemContainingTextExists(
-            totalCookieProtectionHintMessage,
-            totalCookieProtectionHintLearnMoreLink,
-            exists = isDisplayed,
-        )
-        assertItemWithDescriptionExists(
-            totalCookieProtectionHintCloseButton,
-            exists = isDisplayed,
-        )
+    fun verifyCookiesProtectionHintIsDisplayed(composeTestRule: HomeActivityComposeTestRule, isDisplayed: Boolean) {
+        if (isDisplayed) {
+            composeTestRule.onNodeWithTag("tcp_cfr.message").assertIsDisplayed()
+            composeTestRule.onNodeWithTag("tcp_cfr.action").assertIsDisplayed()
+        } else {
+            composeTestRule.onNodeWithTag("tcp_cfr.message").assertDoesNotExist()
+            composeTestRule.onNodeWithTag("tcp_cfr.action").assertDoesNotExist()
```

Comment:
The check that verifies that the "dismiss" button is not displayed, seems to be missing.
`composeTestRule.onNodeWithTag("cfr.dismiss").assertDoesNotExist()`